crates/rolldown/src/bundler.rs

```diff
@@ -254,6 +261,7 @@ impl Bundler {
     }
 
     let mut output = bundle_output?;
+    self.assets.extend(output.assets.iter().map(Clone::clone));
```

Comment:
This line might get triggered many times due to incremental build.

Reply:
Something else should own the assets, I stored them in bundler to prove my hypothesis.